charts/argocd-notifications/values.yaml

```
@@ -35,6 +35,17 @@ secret:
       token:
       # Optional override username
       username:
+      # Optional override icon
+      icon:
+      # Optional override signingSecret: https://argoproj-labs.github.io/argocd-notifications/recipients/slack-bot/
+      signingSecret:
+
+    grafana:
+      enabled: false
+      # https://grafana.example.com/api
+      apiUrl:
+      # Grafana api key
+      apiKey:
```

Would like to see this preserve comment structure:
```suggestion
    grafana:
      # For more information: https://argoproj-labs.github.io/argocd-notifications/services/grafana/
      
      # Specifies whether Grafana notifier should be configured
      enabled: false
      # Grafana api endpoint; for example: https://grafana.example.com/api
      apiUrl:
      # Grafana api key
      apiKey:
```